frameworks/Ruby/grape/config.ru

```diff
@@ -3,25 +3,26 @@ require 'active_record'
 
 Bundler.require :default
 
-db_config = YAML.load(ERB.new(File.read("config/database.yml")).result)[ENV['RACK_ENV']]
+db_config = YAML.load(ERB.new(File.read('config/database.yml')).result)[ENV['RACK_ENV']]
 ActiveRecord::Base.establish_connection(db_config)
 
 class World < ActiveRecord::Base
-  self.table_name = "World"
+  self.table_name = 'World'
 end
 
 module Acme
   class HelloWorld < Grape::API
     get '/json' do
-      {message:"Hello, World!"}
+      {message:'Hello, World!'}
     end
   end
 
   class PlainText < Grape::API
-    content_type :plain, "text/plain; charset=utf-8"
+    content_type :plain, 'text/plain'
```

Comment:
There seems to be a bit of ambiguity here. The requirements say the following:

```
The response content type must be set to text/plain.
```
by the example response is this:

```
HTTP/1.1 200 OK
Content-Length: 15
Content-Type: text/plain; charset=UTF-8
Server: Example
Date: Wed, 17 Apr 2013 12:00:00 GMT

Hello, World!
```

I guess it depends if the smoke tests pass.